pkg/securitypolicy/securitypolicy.go

```diff
@@ -127,6 +131,11 @@ type EnvRule struct {
 	Rule     string     `json:"rule"`
 }
 
+type ExpectedMounts struct {
+	Length   int               `json:"length"`
```

Comment:
why do we need this?

Reply:
again in case anyone else is curious about this it's because SKR doesn't fully support arrays as of now so we need to provide a way to get the length